govee_api_laggat/api.py

```diff
@@ -221,34 +223,8 @@ def rate_limit_on(self, val):
     async def check_connection(self) -> bool:
         """Check connection to API."""
         # TODO: ping is removed from API, get devices list here!
-        try:
-            # this will set self.online
-            await self.ping()
-        except:
-            pass
         return self._govee.online
```

Comment:
in future we probably want to remove this as breaking change, as the api doesn't support this anymore.